doc/guide/readme.md

```diff
@@ -1,57 +1,132 @@
-# An Introduction to Gerbil
+# Guide
+
 This is a quick introductory guide to Gerbil for seasoned schemers;
 it assumes familiarity with scheme and exposure to a couple of
 different implementations.
 
 In the following `$` is the shell prompt and `>` the gxi
 interpreter prompt.
+## Installation
```

Comment:
Not sure I want to include the installation instructions in this document.

Reply:
I think they should be on a separate `install.md` document and call this `guide/intro.md` keeping the title as an "Introduction to Gerbil"